projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/handlers.py

```diff
@@ -165,6 +166,8 @@ class CreateDeploymentHandler(APIHandler):
     def post(self):
         input_data = self.get_json_body()
         try:
+            job = NotebookJobDirectory(input_data[VdkOption.PATH.value])
+            job.remove_outputs_from_all()
```

Comment:
I don't think it's good idea to remove the outputs of the local jupyter notebook. 

That would be rather surprising to the user. He click deploys and everything gets wiped out . 

Instead only the deployed version should be without outputs.

Reply:
i have changed the implementation to work with a copy of the job